Improve manifest information (#93)
SAP-samples/ui5-typescript-walkthrough: steps/01/README.md

### webapp/manifest.json \(New\)

The manifest serves as a crucial configuration file for applications, components, and libraries. Stored in the `webapp` folder, this file is read by OpenUI5 to instantiate a component. Although we haven't created a component yet (which is part of [Step 9: Component Configuration](../09/README.md)), creating the manifest now is necessary due to the UI5 Tooling requirements for development.

Let's start by creating a new file named `manifest.json` in the webapp folder and define its essential attributes:

-   The `_version` attribute in the root is mandatory in the manifest. It indicates the format version of the manifest, which is crucial for identifying application settings whenever the file is read by various tools. As new features or changes are introduced in future versions, maintaining the correct version number ensures compatibility and precise interpretation of the manifest's contents. 

> Since we want the app we build to supports not only the latest OpenUI5 version but also the latest long-term version, which is OpenUI5 1.120, we set the descriptor format version to `1.60.0`.

Reviewer: ```suggestion
Since we want our app to support not the latest OpenUI5 version but rather the latest long-term maintenance version, which is OpenUI5 1.120, we set the format version to `1.60.0`.
```

  > 📝  **Note:** <br>
  > The manifest version should not necessarily align directly with the OpenUI5 version being used. Instead, choose the descriptor version that matches the requirements or supports the features you intend to use in your application. 
  

>   To find the appropriate `_version` for each OpenUI5 release, refer to [Descriptor for Applications, Components, and Libraries \(manifest.json\)](https://sdk.openui5.org/topic/be0cf40f61184b358b5faedaec98b2da.html).

Reviewer 2: Just FYI: the information is also available here: https://github.com/SAP/ui5-manifest/blob/main/mapping.json

>   To find the appropriate `_version` for each OpenUI5 release, refer to [Descriptor for Applications, Components, and Libraries \(manifest.json\)](https://sdk.openui5.org/topic/be0cf40f61184b358b5faedaec98b2da.html) or the [mapping.json](https://github.com/SAP/ui5-manifest/blob/main/mapping.json) file of the [ui5-manifest](https://github.com/SAP/ui5-manifest/) project.

Reviewer 2: The name of the topic in the documentation will be also adopted in the future to manifest, right?

Author: Just checked and this is currently the title of the chapter.
@KlattG let us know in case this is changed.

Reviewer: The topic has recently been renamed to "Manifest (Descriptor for Applications, Components, and Libraries)".
I hope that's fine with you.

-   The **`sap.app`**  namespace defines properties specific to the application. It includes the following required attributes:

>     -   `id`: This specifies an identifier for the application using dot notation, limited to 70 characters. It must be unique and must correspond to the component ID/namespace.

Reviewer: ```suggestion
    -   `id`: This specifies an identifier for the application. It is provided in dot notation and is limited to 70 characters. It must be unique and must correspond to the component ID/namespace.
```

>     -   `type`: This property helps in identify whether the project is an `application` or `component`. Including it provides a useful classification and assists in the correct loading of the application.

Reviewer: ```suggestion
    -   `type`: This property helps to identify whether the project is an `application` or a `component`. It provides a useful classification and assists in the correct loading of the application.
```

>     -   `type`: This property helps in identify whether the project is an `application` or `component`. Including it provides a useful classification and assists in the correct loading of the application.

Reviewer 2: To be precise, it can be also "library" or "card"
